samples/cpp/benchmark_app/utils.cpp

```diff
@@ -164,26 +160,17 @@ size_t getBatchSize(const benchmark_app::InputsInfo& inputs_info) {
     return batch_size;
 }
 
-InferenceEngine::Layout getLayoutFromString(const std::string& string_layout) {
-    static const std::unordered_map<std::string, InferenceEngine::Layout> layouts = {
-        {"NCHW", InferenceEngine::Layout::NCHW},
-        {"NHWC", InferenceEngine::Layout::NHWC},
-        {"NCDHW", InferenceEngine::Layout::NCDHW},
-        {"NDHWC", InferenceEngine::Layout::NDHWC},
-        {"C", InferenceEngine::Layout::C},
-        {"CHW", InferenceEngine::Layout::CHW},
-        {"HWC", InferenceEngine::Layout::HWC},
-        {"HW", InferenceEngine::Layout::HW},
-        {"NC", InferenceEngine::Layout::NC},
-        {"CN", InferenceEngine::Layout::CN}};
-    auto it = layouts.find(string_layout);
-    if (it != layouts.end()) {
-        return it->second;
+size_t getFunctionInputBatchSize(const ov::Model& function) {
+    try {
+        auto& param = function.get_parameters()[0];
+        auto layout = param->get_layout();
+        return param->get_shape().at(ov::layout::batch_idx(layout));
+    } catch (...) {
+        return 1;  // Defualt batch value
     }
-    IE_THROW() << "Unknown layout with name '" << string_layout << "'.";
 }
 
-std::string getShapeString(const InferenceEngine::SizeVector& shape) {
+std::string getShapeString(const ov::Shape& shape) {
```

Comment:
ov::Shape has operator <<. I think you don't need this method.

Reply:
still required, as there's no method to get std::string, as we need it in some parts of code. function code is modified to use << operator internally.